Runtime/Core/Shared/Response.cs

```diff
@@ -27,5 +27,20 @@ public class Response
             response.Success = false;
             return response;
         }
+
+        private Response(string errorCode, string errorMessage = "")
+        {
+            ErrorCode = errorCode;
+            ErrorMessage = errorMessage;
+        }
+        
+        public Response()
+        {
+        }
+        
+        public static Response Fail(string errorCode, string errorMessage = "")
+        {
```

Comment:
You should also set `response.Success = false` in case someone is reading this value.

Reply:
Done